site/content/admin-guide/common-config.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "Common Configuration"
+weight = 300
++++
+
+Most Hono components support a common set of configuration options. This section
+describes those options. <!--more-->
+
+Each component which supports the following options explicitly states so. If
+it doesn't, then these options are not supported by this component.
+
+## Using epoll() on Linux 
+
+Netty allows the use of `epoll()` on Linux x86_64 based systems. Hono provides
```

Comment:
IMHO it would be good to explain what `epoll` is about and/or provide a link to a resource where I can learn about it. I do not expect many people to actually know why they should activate/use this feature without further explanation.

Reply:
I added a short section about that.